letsencrypt/models/acme_tiny.py

```diff
@@ -0,0 +1,199 @@
+# flake8: noqa
```

Comment:
`acme_tiny.py` is a lightweight client for Let's Encrypt (replacing the official one) which can be found here: https://github.com/diafygi/acme-tiny , it should not be changed.